pkg/model/model.go

```diff
@@ -107,6 +107,17 @@ type Model interface {
 	CreateMetadataConfiguration(ctx context.Context, metadata *MetadataConfiguration) error
 	GetMetadataConfiguration(ctx context.Context, repoDID string) (*MetadataConfiguration, error)
 	DeleteMetadataConfiguration(ctx context.Context, repoDID string) error
+
+	CreateModerationDelegation(ctx context.Context, delegation *ModerationDelegation) error
+	DeleteModerationDelegation(ctx context.Context, rkey string) error
+	GetModerationDelegation(ctx context.Context, streamerDID, moderatorDID string) (*ModerationDelegation, error)
+	GetModerationDelegations(ctx context.Context, streamerDID, moderatorDID string) ([]*ModerationDelegation, error)
+	GetModeratorDelegations(ctx context.Context, moderatorDID string) ([]*ModerationDelegation, error)
+	GetStreamerModerators(ctx context.Context, streamerDID string) ([]*ModerationDelegation, error)
+
+	CreateAuditLog(ctx context.Context, log *ModerationAuditLog) error
+	GetAuditLogs(ctx context.Context, streamerDID string, limit int, before *time.Time) ([]*ModerationAuditLog, error)
+	GetModeratorAuditLogs(ctx context.Context, moderatorDID string, limit int, before *time.Time) ([]*ModerationAuditLog, error)
```

Comment:
Still reading through but I don't think these can be in `model` -- that exists on every Streamplace node in parallel and should be mostly for records indexed from atproto. If we have secret multinode state that needs to go in `statefuldb` instead so it can be shared with multiple nodes through Postgres